modules/contrib/default_content/tests/modules/default_content_test/default_content_test.info.yml.bak

```diff
@@ -0,0 +1,14 @@
+name: 'Default content tests'
```

Comment:
is this .bak file required?

Reply:
@giteshk default_content_test.info.yml.bak is coming from contrib module default_content. Once this is added to the main composer file along with the rest of contrib modules required, we can remove all contrib modules from the repo.

Reply:
@kiranjyoth can we remove all the contrib modules/themes/libraries and just have the composer.json and composer.lock file and any custom code edits in this PR ?

Reply:
I will update the PR to reflect these changes. Thanks @giteshk